pkgs/data/fonts/cozette/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchurl, mkfontscale }:
+
+let
+  version = "1.5.1";
+  releaseUrl =
+    "https://github.com/slavfox/Cozette/releases/download/v.${version}";
+in stdenv.mkDerivation rec {
+  inherit version;
+  pname = "Cozette";
```

Comment:
```suggestion
  pname = "Cozette";
  inherit version;

```

Reply:
Cool, right on.  Makes sense.  I have another change to add as well.

Reply:
That'll do.